# 
# Submitted-By: Robert Schwebel, 2005-05-25
# Committed-By: Robert Schwebel
#
# Error: 
#
# arm-softfloat-linux-gnu-gcc -O -I../include  -Wall -isystem
# /ptx/work/svn-rsc/ptxdist-0.7-trunk/local/arm-softfloat-linux-gnu/include
# -c -o tst_kvercmp.o tst_kvercmp.c
# tst_kvercmp.c:44:29: macro "strsep" requires 2 arguments, but only 1 given
# tst_kvercmp.c: In function `get_kver':
# tst_kvercmp.c:44: error: `strsep' redeclared as different kind of symbol
#
# Description: 
#
# Re-declaration of strcmp; we just remove that. 
#
# State:
# 
# no real fix, so not being sent upstream
#

diff -urN ltp-full-20050505/lib/tst_kvercmp.c ltp-full-20050505-ptx/lib/tst_kvercmp.c
--- ltp-full-20050505/lib/tst_kvercmp.c	2005-05-05 21:16:10.000000000 +0200
+++ ltp-full-20050505-ptx/lib/tst_kvercmp.c	2005-05-25 11:33:06.000000000 +0200
@@ -41,7 +41,6 @@
 	struct utsname uval;
 	char *kver;
 	char *r1, *r2, *r3;
-	extern char *strsep();          /* shut up some compilers */
 
 	uname(&uval);
 	kver = uval.release;
